kfs-core/src/main/java/edu/arizona/kfs/fp/batch/service/impl/ProcurementCardCreateDocumentServiceImpl.java

```diff
@@ -79,7 +79,7 @@ public boolean rerouteProcurementCardDocuments() {
 
         for (ProcurementCardDocument pcardDocument : documents) {
         
-            String pcardDocumentId = pcardDocument.getObjectId();
+        	String pcardDocumentId = pcardDocument.getDocumentNumber();
```

Comment:
Spacing fixed. As for the pause, it looks like it was added to keep the job from failing (KFSI-2969). Changed  InterruptedException to be logged instead of throwing a RuntimeException.